output/file.go

```diff
@@ -0,0 +1,101 @@
+package output
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/vbauerster/mpb/v5"
+	"github.com/vbauerster/mpb/v5/decor"
+)
+
+type FileWriter struct {
+	FullPath string
+	Filename string
```

Comment:
`Filename` is always calculated by `filepath.Base(FullPath)`.
So I think that keeping `Filename` in the struct is redundant.
Please remove `Filename` from struct and create new method `Filename()` to calculate filename.

Reply:
`FullPath` is not used outside of this module.
So `FullPath` can be private.

Reply:
Done.